benchmarks/benchmark_exposure.py

```diff
@@ -12,8 +12,15 @@ class ExposureSuite:
     def setup(self):
         self.image = img_as_float(data.moon())
         self.image = rescale(self.image, 2.0, anti_aliasing=False)
+        # for Contrast stretching
+        self.p2, self.p98 = np.percentile(self.image, (2, 98))
 
     def time_equalize_hist(self):
-        # Running it 10 times to achieve significant performance time.
-        for i in range(10):
-            result = exposure.equalize_hist(self.image)
```

Comment:
why did you get rid of the for loop?

Reply:
@emmanuelle turns out asv does this kind of averaging by default.